query-connector/src/app/ui/components/header/header.tsx

```diff
@@ -124,41 +124,51 @@ const HeaderComponent: React.FC<{ authDisabled: boolean }> = ({
             )}
           </div>
         </div>
+        {showMenu && (
+          <div ref={menuRef} className={styles.menuDropdownContainer}>
+            <ul
+              id="dropdown-menu"
+              className={classNames(`usa-nav__submenu`, styles.menuDropdown)}
+            >
+              {/* TODO: Enable this once we can show/hide rules based on actual auth status */}
+              {/* {isProduction && ( */}
+              <>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={"/queryBuilding"}>
+                    My queries
+                  </Link>
+                </li>
+                <li className={styles.subMenuItem}>
+                  <Link className={styles.menuItem} href={"/fhir-servers"}>
+                    FHIR Servers
```

Comment:
also worth aligning on whether we want Sentence case or Title Case for these. Don't have a strong preference either way (though maybe check with design in case they do?) as long as they're aligned.

Reply:
I have made the change as it was small enough for the `My Queries` menu item.